commands/create_command.py

```diff
@@ -0,0 +1,23 @@
+from Icommand import ICommand
+import os, json, sys
+parent_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
+sys.path.append(parent_dir)
+from SchemaKeys import SchemaKeys
+
+class CreateCommand(ICommand):
+    def __init__(self, schema):
+        self.schema = schema
+        self.data = json.load(open(schema, "r"))
+
+    def execute(self):
+        if (self.schema == None or not os.path.exists(os.path.join(parent_dir, self.schema))):
+            raise Exception("FileNotFound")
+
+        dir = self.data[SchemaKeys().DATABASE]
+        path = os.path.join(parent_dir, dir)
+        
+        os.makedirs(path, exist_ok = True)
+
+        for table in data[SchemaKeys().TABLES]:
```

Comment:
Why do you create an instance of `SchemaKeys()` everytime. This class should be an abstract class and we shouldn't create any instace of it. 
we should call the variable by `SchemaKeys.TABLES`